.github/workflows/regular-release.yaml

```diff
@@ -1,66 +1,83 @@
 on:
-  schedule:
```

Comment:
The changes in this commit seem unrelated to the PR title, though I can see how its all related... Could you either (a) change the PR title to make it clear that this PR does more than just add instructions to documentation, or (b) move this commit to a separate PR. I'd personally suggest (a) :-)

Reply:
I implemented (a) 👍